### PR TITLE
Add Privacy Policy

### PR DESCRIPTION
**Pull request by arturi**

### CODE REVIEW

File: src/pages/privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+We keep a record of the traffic data that is logged automatically by our
+servers, such as your Internet Protocol (IP) address, device information, the
+website that you visited before ours and the website you visit after leaving our
+site. We also collect some site, application and service statistics, such as
+access rates, page hits and page views. We are not able to identify any
+individual from traffic data or site statistics.
+
+#### Google Analytics
+
+Our platform uses Google Analytics, which uses cookies. The information created
```
Comment (aduh95) [inline src/pages/privacy-policy.md:149]: AFAIK that's simply not true, or am I missing something?

File: src/pages/privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+your device. We use cookies to help us provide you with a personalized service,
+and to help make our website and service better for you.
+
+Our cookies may be session cookies (temporary cookies that identify and track
+users within our websites) which are deleted when you close your browser, or
+persistent cookies (cookies that enable our website to “remember” who you are
+and to remember your preferences within our website) which will stay on your
+computer or device after you close your browser).
+
+We use the following different types of cookies:
```
Comment (aduh95) [inline src/pages/privacy-policy.md:117]: I think Uppy.io uses no cookies 🤔

File: src/pages/privacy-policy.md (hunk 1 above)
Comment (arturi) [reply src/pages/privacy-policy.md:149]: True! I copy-pasted, this should be changed! Opening another PR with a fix. Thanks for the find!

File: src/pages/privacy-policy.md (hunk 2 above)
Comment (arturi) [reply src/pages/privacy-policy.md:117]: Also true!

**Pull request merged**